src/components/ManagerPage/AppsList.js

```diff
@@ -43,27 +44,17 @@ const ICONS_FALLBACK = {
 type Status = 'loading' | 'idle' | 'busy' | 'success' | 'error'
 type Mode = 'home' | 'installing' | 'uninstalling'
 
-type LedgerApp = {
-  name: string,
-  version: string,
-  icon: string,
-  app: Object,
-  bolos_version: {
-    min: number,
-    max: number,
-  },
-}
-
 type Props = {
   device: Device,
   targetId: string | number,
   t: T,
+  version: string,
 }
 
 type State = {
   status: Status,
   error: string | null,
-  appsList: LedgerApp[],
+  appsList: LedgerScriptParams[] | Array<*>,
```

Comment:
is it intended to have uncertain type here?

Reply:
it was bitchin if my array was empty...

Reply:
very strange :/ do you still reproduce?